web_client/src/components/DataImportExport.vue

```diff
@@ -80,7 +80,8 @@ export default defineComponent({
         } else {
           response = await djangoRest.projectExport(currentProject.value.id);
         }
-        if (response.warnings) {
+        console.log(response);
```

Comment:
```suggestion
```

Reply:
Changed in [c2646a4](https://github.com/OpenImaging/miqa/pull/563/commits/c2646a4d439639257076c7cb52a46a2148540dc3)